shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -170,6 +171,7 @@ - (void)triggerTouchRateCorrectionIfNeeded:(NSSet*)touches;
 @interface FlutterViewControllerTest : XCTestCase
 @property(nonatomic, strong) id mockEngine;
 @property(nonatomic, strong) id mockTextInputPlugin;
+@property(nonatomic, strong) id mockHourCycle;
```

Comment:
For clarity, consider naming this `mockHourFormat`.

Reply:
This is actually not used/needed/